examples/math/trigonometryExample/src/ofApp.cpp

```diff
@@ -146,27 +145,25 @@ void ofApp::keyReleased(int key){
 void ofApp::mouseMoved(int x, int y ){
 
 }
-
+//--------------------------------------------------------------
+void ofApp::checkMouse(float x, float y){
```

Comment:
Is this new method just to reduce repetitive code?

Reply:
yes.